syft/frameworks/torch/tensors/interpreters/replicative_shared.py

```diff
@@ -0,0 +1,65 @@
+import torch
+from syft.generic.abstract.tensor import AbstractTensor
+import random
+
+
+class ReplicatedSharingTensor(AbstractTensor):
+    def __init__(
+        self, shares=None, owner=None, id=None, tags=None, description=None,
+    ):
+        super().__init__(id=id, owner=owner, tags=tags, description=description)
+        self.child = shares
+        self.ring_size = 2 ** 5
+
+    def share_secret(self, secret, workers):
+        number_of_shares = len(workers)
+        shares = self.generate_shares(secret, number_of_shares)
+        shares_locations = self.distribute_shares(workers, shares)
+        self.child = shares_locations
+        return self
+
+    def generate_shares(self, secret, number_of_shares=3):
```

Comment:
MAYBE: make this ```staticmethod``` and test it with more shares?

Reply:
I don't think this can be a static method because it uses an instance variable (self.ring_size), but we can still test it on more shares

Reply:
PICKY: This is meant to be called only from inside the ```ReplicatedSharingTensor``` class, right? (in that case maybe we can prefix the method name with ```_``` such that we know it is internal?